spec/rubocop/cop/rspec/redundant_predicate_matcher_spec.rb

```diff
@@ -0,0 +1,132 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::RSpec::RedundantPredicateMatcher do
+  it 'registers an offense when using `be_all`' do
+    expect_offense(<<~RUBY)
+      expect(foo).to be_all be_odd
+                     ^^^^^^^^^^^^^ Use `all` instead of `be_all`.
+    RUBY
+
+    expect_correction(<<~RUBY)
+      expect(foo).to all be_odd
+    RUBY
+  end
```

Comment:
This may deserve its own cop:
```
/Users/pirj/source/real-world-rspec/gitlabhq/spec/services/deployments/archive_in_project_service_spec.rb:29:49: C: [Correctable] RSpec/RedundantPredicateMatcher: Use all instead of be_all.
        expect(deployments.map(&:archived?)).to be_all(false)
                                                ^^^^^^^^^^^^^
```
It could be `expect(deployments).to all be_archived(false)`. Looks bad with such an informal negation, though.

Reply:
This looks like a good jurisdiction for other cops.